recipe/meta.yaml

```
@@ -27,24 +26,36 @@ requirements:
     - nodejs >=18
```

For those of us who need a bit of help, we might point to the reason why we add these two:

```suggestion
    # nodejs/esbuild required, see https://github.com/panel-extensions/panel-graphic-walker/blob/main/DEVELOPER_GUIDE.md
    - nodejs >=18
```